README.md

```diff
@@ -60,3 +60,15 @@ The `solid-client-authn` libraries are compatible with [NSS](https://github.com/
 
 - [Using Inrupt Solid JavaScript Client Libraries to authenticate](https://docs.inrupt.com/developer-tools/javascript/client-libraries/tutorial/authenticate/)
 - [Inrupt documentation Homepage](https://docs.inrupt.com/)
+
+## How to run the test client login app?
+
+```javascript
```

Comment:
Not JavaScript though 😉

Reply:
Yep, should be ```shell

Reply:
Oops I used bash, but you're right, shell is more generic. I didn't know it was available.